OrrRock/OrrRock/Extensions/UIImage+.swift

```diff
@@ -13,4 +13,77 @@ extension UIImage {
             draw(in: CGRect(origin: .zero, size: size))
         }
     }
+    
+    // https://stackoverflow.com/questions/158914/cropping-an-uiimage
+    func cropped(rect: CGRect) -> UIImage? {
+        if let image = self.cgImage!.cropping(to: rect) {
+            return UIImage(cgImage: image)
+        } else if let image = (self.ciImage)?.cropped(to: rect) {
+            return UIImage(ciImage: image)
+        }
+        return nil
+    }
+    
+    func imageRectAs16to9() -> CGRect  {
+        let originalHeight = self.size.height
+        let originalWidth = self.size.width
+
+        let resizedWidth = originalHeight * 9 / 16
+        let originXCoordinate = (originalWidth - resizedWidth)/2
+        let rect = CGRect(x: originXCoordinate, y: 0, width: resizedWidth, height: originalHeight)
+        
```

Comment:
세로가 긴 경우에도 대응이 필요합니다!

```suggestion
        if originalWidth * 16 / 9 < originalHeight {
        // 16+@ : 9 사이즈인 경우
            let resizedHeight = originalWidth * 16 / 9
            let originYCoordinste = (originalHeight - resizedHeight) / 2
            let rect = CGRect(x: 0, y: originYCoordinste, width: originalWidth, height: resizedHeight)
        } else {
        // 16 : 9+@ 사이즈인 경우
            let resizedWidth = originalHeight * 9 / 16
            let originXCoordinate = (originalWidth - resizedWidth)/2
            let rect = CGRect(x: originXCoordinate, y: 0, width: resizedWidth, height: originalHeight)
        }
```

Reply:
감사합니다 레드 @sm-amoled ! 따로 이슈를 생성해서 수정해야할지 고민하고 있었는데...!!! 너무나도 좋은! ☺️0c24ca0f3d12447611d3720047730d52c1670e34 에 반영했습니다! ☺️🐈☺️